src/ctype/CType.ts

```diff
@@ -18,6 +18,18 @@ import { getOwner, store } from './CType.chain'
 import TxStatus from '../blockchain/TxStatus'
 import IClaim from '../types/Claim'
 
+export function compressCTypeSchema(cTypeSchema: ICType): any {
+  const { schema } = cTypeSchema
+  return [schema.$id, schema.$schema, schema.properties, schema.type]
+}
+
+export function compressCType(cType: ICType): any {
+  if (cType.owner) {
```

Comment:
Change request:
You can simplify/shorten this and get rid of your `if`: 
`return [compressCTypeSchema(cType), cType.owner || null, cType.hash]`

Reply:
What Timo has mentioned, he wants the code to be more verbose. However, I do think this still details what is happening.

Reply:
Even more concise now 👍